### DISCUSSION TITLE
@xstate/test only generate paths to final states

### DISCUSSION THREAD

**Discussion by UberMouse** [statelyai/xstate#2782] [Q&A]
From looking at the @xstate/test API it doesn't seem like it's possible to automatically only generate test paths to final states? I tend to write test models where this is all I'm trying to achieve, but by default using `getSimplePathPlans` @xstate/test will end up with redundant test plans that terminate in intermediate states that get tested by a path to final states.

Currently what I end up doing is just manually creating the paths via calling `getSimplePathPlansTo` for each of the final state nodes in the model. Is there something in @xstate/test I'm missing or should I look into creating a PR to add something like `getSimplePathPlansToFinalStates` or adding an option to `getSimplePathPlans`?

**Comment by davidkpiano** [comment]
The option doesn't exist right now, but I agree that it should.
